README.md

```diff
@@ -1,8 +1,8 @@
 <!--
 [![npm-version](https://img.shields.io/npm/v/CloudEngineering/node-sdk-template.svg)](https://www.npmjs.com/package/node-sdk-template)
 -->
-# DPX Services Node.js SDK
-Node.js client library to interact with various [DPX APIs](https://cloud.ibm.com/apidocs?category=<service-category>).
+# Data Product Exchange Node.js SDK
+Node.js client library to interact with various [Data Product Exchange Service APIs](https://cloud.ibm.com/apidocs?category=<service-category>).
```

Comment:
```suggestion
Node.js client library to interact with various Data Product Exchange Service APIs.
```
If/when the appropriate service category is identified, add back the link.  But for now, it's probably better to omit the link since it's not correct.

Reply:
Updated